src/ResourceManager/ContainerInstance/Commands.ContainerInstance/Commands/NewAzureContainerGroupCommand.cs

```diff
@@ -149,6 +177,34 @@ public class NewAzureContainerGroupCommand : ContainerInstanceCmdletBase
         [ValidateNotNullOrEmpty]
         public PSCredential RegistryCredential { get; set; }
 
+        [Parameter(
+            Mandatory = true,
+            ParameterSetName = CreateContainerGroupWithAzureFileVolumeParamSet,
+            HelpMessage = "The name of the Azure File share to mount.")]
+        [ValidateNotNullOrEmpty]
+        public string AzureFileVolumeShareName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ParameterSetName = CreateContainerGroupWithAzureFileVolumeParamSet,
+            HelpMessage = "The storage account name of the Azure File share to mount.")]
+        [ValidateNotNullOrEmpty]
+        public string AzureFileVolumeAccountName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ParameterSetName = CreateContainerGroupWithAzureFileVolumeParamSet,
+            HelpMessage = "The storage account key of the Azure File share to mount.")]
+        [ValidateNotNullOrEmpty]
+        public PSCredential AzureFileVolumeAccountKey { get; set; }
```

Comment:
Is account key the correct name for this parameter? I wouldn't usually consider a PSCredential to be a "key".  Please correct me if I am wrong.

Reply:
Are you suggesting to have something `PSCredential AzureFileVolumeAccountCredential` and have username and key in it?

Reply:
That's what I would consider a PSCredential to be - it requires both a username and password, so "key" seems like the wrong name.